client/lib/billing/app/views/overlays/billing.js

```diff
@@ -0,0 +1,12 @@
+import OverlayView from 'tahi/views/overlay';
+
+export default OverlayView.extend({
+  templateName: 'overlays/billing',
+  layoutName:   'layouts/overlay',
+  cardName: 'billing',
+
+  didInsertElement: function() {
+    var choice = $(".payment-method .select2-container").select2("val");
+    this.get("controller").set("selectedPaymentMethod", choice);
```

Comment:
I think this can get fixed once billing card starts persisting data. I am assuming it's not persisting the data right now.